DocumentedExamples/SeaIce_Obs_Model_Compare.ipynb

```diff
@@ -0,0 +1,5185 @@
+{
```

Comment:
<p>why not use timeseries in <code>/g/data3/hh5/tmp/cosima/observations/NOAA/G02135/south/monthly/data/*.csv</code>  as in <a href="https://github.com/COSIMA/ACCESS-OM2-1-025-010deg-report/blob/master/figures/ice_timeseries/ice_timeseries.ipynb" target="_blank">https://github.com/COSIMA/ACCESS-OM2-1-025-010deg-report/blob/master/figures/ice_timeseries/ice_timeseries.ipynb</a> ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/311/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
<p>It is convenient to use the CDR gridded Sea Ice Concentration because its used for the section titled "Sea Ice Area differences" (which I just realised needs a better name). It also ensures that the method for calculation of sea-ice area is consistent between the model results and observations. (I'm 95% confident this is cell is how the Sea Ice Index / G02135 is calculated but sometimes there weird quirks).</p>

Reply:
<p>ok fair enough</p>